src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -356,44 +357,58 @@ BOOST_AUTO_TEST_CASE(GradientReconstructionLossTest)
 }
 
 /*
- * Simple test for the dice loss function.
```

Comment:
Hi @shardulparab97, I see you have removed dice loss function. I guess either it is done mistakenly or is there any particular reason for removing it?

Reply:
Hi @manish7294, when I was working on this PR, the dice loss was not yet merged into the master.
I used the code in that PR as a reference, so hence it has come up like this :)

Reply:
That explains it. Maybe you could make the required change here :)